WordPress/Classes/ViewRelated/People/PeopleViewController.swift

```diff
@@ -70,6 +70,8 @@ class PeopleViewController: UITableViewController, UIViewControllerRestoration {
         //
         switch filter {
         case .followers:
+            fallthrough
+        case .email:
```

Comment:
[nit]

imo `case .followers, .email:` seems more readable

Reply:
@wargcm wdyt? Double-checking just in case.

Reply:
It's a good call, it was something I overlooked. `fallthrough` probably makes more sense if you're combining some of the cases whereas this is just two cases with the same result.